src/element/Nip05.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheck } from "@fortawesome/free-solid-svg-icons";
+
+import './Nip05.css'
+
+const Nip05 = ({ nip05, pubkey }) => {
+    const [nip05pubkey, setNip05pubkey] = useState()
+    const isVerified = nip05pubkey === pubkey
+    const [nick, domain] = nip05.split('@')
+
+    useEffect(() => {
+        fetch(`https://${domain}/.well-known/nostr.json`)
+          .then((res) => res.json())
+          .then(({ names }) => {
+            if (names && names[nick]) {
+              setNip05pubkey(names[nick])
+            }
+          })
+          .catch((err) => {
+            console.error("Couldn't verifiy nip05")
+          })
+    }, [nip05, nick, domain])
+
+    return (
+       <div className="flex nip05">
+         <div className="nick">{nick}</div>
+         <div className="domain">@{domain}</div>
+         {isVerified && (
```

Comment:
If its not verified then we should show error right?

Reply:
<img width="282" alt="Screenshot 2023-01-10 at 07 19 08" src="https://user-images.githubusercontent.com/92299987/211476232-a81fca20-7a0b-4774-b914-8e449a3522c5.png">